operators/src/mock/mock_data_set_data_source.rs

```diff
@@ -0,0 +1,174 @@
+use crate::engine::{
+    ExecutionContext, InitializedOperator, InitializedOperatorImpl, InitializedVectorOperator,
+    LoadingInfo, QueryContext, QueryProcessor, QueryRectangle, SourceOperator,
+    TypedVectorQueryProcessor, VectorOperator, VectorQueryProcessor, VectorResultDescriptor,
+};
+use crate::util::Result;
+use futures::stream;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::collections::{MultiPointCollection, VectorDataType};
+use geoengine_datatypes::dataset::DataSetId;
+use geoengine_datatypes::primitives::{Coordinate2D, TimeInterval};
+use geoengine_datatypes::spatial_reference::SpatialReferenceOption;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Clone)]
+pub struct MockDataSetDataSourceLoadingInfo {
+    pub points: Vec<Coordinate2D>,
+}
```

Comment:
Can't we make it for every FeatureCollection input?

Reply:
yes, as a separate task